### ISSUE TITLE
Export Poll as iCalender file

### ISSUE THREAD

**Issue by ascheman** [nextcloud/polls#1633]
## Describe the goal you'd like to achieve

As a **poll participant**, I want to **be able to access my poll vote and the final result as iCalendar file (stream)**, because **it helps to integrate my possible/final appointments with other personal calendars**.

This should not be a one time export as with #366 but a permanently or temporarily available ics file (like a shared file). Any updates on my poll vote or the final decision of the poll owner should change the contents of the ics file so that my other calendar can continuously synchronise with the poll.

**Comment by dartcafe** [comment]
See #400 and #121. Still a to do.

**Comment by Elvis2000NC** [comment]
yes, that would be a great function. I only want to see the appointments in my calendar automatically which i say "yes". The other appointments, i say "no", should not shown in my calender.

**Comment by 2EricBrunet** [comment]
I'm support this request. I use polls extension in a professional context.
My needs are a little different: 
* integrate all events with a prefix status in the calendar title events. Actually I add a '?' as the prefix with name of the poll as title (when I said Yes).  It's immediatly visual when a see events in all views (day,week,month). In same manner, create 2 others prefix for answer 'no' and answer 'maybe', or better: let's user to choice own prefix
* export file is the minimum, I'm not sure it is a good practice to subscribe a plenty of pools (with a url) for a calendar client (a poll has a limited lifecycle of updates, few weeks in many case).
